apps/test/netsim/NetSimRouterNode.js

```diff
@@ -427,7 +421,11 @@ describe("NetSimRouterNode", function () {
       for (var wireID = routerA.entityID + 1;
            wireID < routerA.entityID + CONNECTION_LIMIT + 1;
            wireID++) {
-        NetSimWire.create(testShard, wireID, routerA.entityID, function () {});
+        NetSimWire.create(testShard, {
+          localNodeID: wireID,
+          remoteNodeID: routerA.entityID,
+          localAddress: wireID.toString(10)
```

Comment:
As of this change, routers reject connections when they detect an address collision.  Here we are making sure each wire has a different address so no collisions occur.